gazelle/manifest/testdata/gazelle_python.yaml

```
@@ -1,13 +1,5 @@
 manifest:
   modules_mapping:
     arrow: arrow
```

Could you please add a test where we have a different distribution providing a namespace package? Something like following would be good to have. We should check that importing arrow.plugin correctly adds arrow_plugin to the deps.
```suggestion
    arrow: arrow
    arrow.plugin: arrow_plugin
```

---

I believe this case will be handled correctly because it is covered in example/build_file_generation.

But it does make sense to add the test. I will add it at night.

---

added